src/Normalizer/StrawberryfieldFieldItemNormalizer.php

```diff
@@ -67,19 +68,42 @@ public function denormalize($data, $class, $format = NULL, array $context = [])
 
     /** @var \Drupal\Core\Field\FieldItemInterface $field_item */
     $field_item = $context['target_instance'];
-    $this->checkForSerializedStrings($data, $class, $field_item);
 
-    $field_item->setValue($this->constructValue($data, $context));
+    $this->checkForSerializedStrings($data, $class, $field_item);
+    
+    // Set each key in the field_item with its constructed value
+    // At this point all values will be encoded JSON strings
+    // Coming from minio data, keys should be 'value' and 'str_flatten_keys'
```

Comment:
Question: 'str_flatten_keys' is a dynamic/calculated field property. Feeding it back makes no sense. If i POST a JSON without it, does this still work? Do i need to make  'str_flatten_keys'  Internal?

Reply:
@DiegoPino `string_flatten_keys` is not needed, the POST works fine without it.
The reason I included it was because it's in the minio dump, and thought that was the test case.